docs/core/install/linux-package-manager-fedora31.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Install .NET Core on Fedora 31 - package manager - .NET Core
+description: Use a package manager to install .NET Core SDK and runtime on Fedora 31.
+author: thraka
+ms.author: adegeo
+ms.date: 12/17/2019
+---
+
+# Fedora 31 Package Manager - Install .NET Core
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]
+
+This article describes how to use a package manager to install .NET Core on Fedora 31. If you're installing the runtime, we suggest you install the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), as it includes both .NET Core and ASP.NET Core runtimes.
+
+## Register Microsoft key and feed
+
+Before installing .NET, you'll need to:
+
+- Register the Microsoft key
+- register the product repository
+- Install required dependencies
+
+This only needs to be done once per machine.
+
+Open a terminal and run the following commands.
+
+```bash
+sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+sudo wget -q -O /etc/yum.repos.d/microsoft-prod.repo https://packages.microsoft.com/config/fedora/31/prod.repo
```

Comment:
The `https://packages.microsoft.com/config/fedora/31/prod.repo` link gives 404 not found.

Reply:
Yes, this is in WIP until I find out if/when it's coming.

Reply:
@Thraka, It's available now. The link is no longer 404.

Reply:
The actual packages will be published around a week from now, then we'll merge this PR.